packages/insomnia-app/app/plugins/index.ts

```diff
@@ -90,6 +90,7 @@ export interface SpecInfo {
 
 export interface ConfigGenerator extends InternalProperties {
   label: string;
+  docsLink: string;
```

Comment:
Should be optional, I think?

```suggestion
  docsLink?: string;
```